### PR TITLE
Added bped and geohash

### PR DESCRIPTION
**Pull request by twist-vector**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -3588,4 +3588,30 @@
...
+    "method": "git",
+    "tags": [
+      "library",
+      "geocoding",
+      "pure"
+    ],
+    "description": "Nim implementation of the geohash latitude/longitude geocode system",
+    "license": "Apache License 2.0",
+    "web": "https://github.com/twist-vector/nim-geohash"
+  }
```
Comment (dom96) [inline packages.json]: Missing a comma here.

**Comment by dom96** [comment]
You can publish using Nimble now btw: ``nimble publish`` should just work (TM).

**Comment by twist-vector** [comment]
That's awesome.  I was just going to ask if there was a plan for that!

Not sure what went wrong.  I created the personal token and that part
seems to have worked, but then it fails on checkout...

Successfully verified as twist-vector
Cloning packages into: /tmp/nimble-packages-fork
Cloning into '/tmp/nimble-packages-fork'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights and the repository exists.
Traceback (most recent call last)
nimble.nim(990)          nimble
nimble.nim(977)          doAction
publish.nim(140)         publish
tools.nim(20)            doCmd

Error: unhandled exception: Execution failed with exit code 32768 [NimbleError]


On Tue, Nov 24, 2015 at 6:12 PM, Dominik Picheta
<notifications@github.com> wrote:
> You can publish using Nimble now btw: nimble publish should just work (TM).
>
> —
> Reply to this email directly or view it on GitHub.



-- 
---
Tom Krauss
thomas.krauss@gmail.com
---
Can of worms?  The can is open, the worms are everywhere.

**Comment by dom96** [comment]
You haven't set up an SSH key for your git.

**Comment by twist-vector** [comment]
That was it.  Thanks.  I need to understand Github better.  It seems to
offer a ton of capabilities.

On Wed, Nov 25, 2015 at 6:07 AM, Dominik Picheta <notifications@github.com>
wrote:

> You haven't set up an SSH key for your git.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nim-lang/packages/pull/279#issuecomment-159573824>.
>



-- 
---
Tom Krauss
thomas.krauss@gmail.com
---
Can of worms?  The can is open, the worms are everywhere.

**Pull request merged**